index.bs

```diff
@@ -727,6 +729,8 @@ To <dfn>normalize element name</dfn> |name|, run these steps:
   1. Return `null`.
 </div>
 
+  Issue(145): we shouldnt normalize lowercase/uppercase for element names at all. Other APIs are also case-sensitive (e.g., MutationObserver)
```

Comment:
Added the full stop. Let's discuss in the issue itself. I've toned down the note into "we should find out" instead of "we shouldnt normalize", for the sake of getting this rather large collection of changes ready to land